io/zenoh-links/zenoh-link-tls/src/unicast.rs

```diff
@@ -518,6 +518,16 @@ impl TlsServerConfig {
                 .map_err(|e| zerror!(e))
                 .map(|mut keys| keys.drain(..).map(PrivateKey).collect())?;
 
+        if keys.len() == 0 {
+            keys = rustls_pemfile::pkcs8_private_keys(&mut Cursor::new(&tls_server_private_key))
+                .map_err(|e| zerror!(e))
+                .map(|mut keys| keys.drain(..).map(PrivateKey).collect())?;
+        }
+
+        if keys.len() == 0 {
+            return Err(zerror!("No private key found.").into());
```

Comment:
This whole line can be shortened to `bail!("No private key found")` :)

Reply:
I replaced the return statement according to your suggestion. Also I fixed a failed CI check so now it passes the CI.